web-frontend/src/pages/Assets/DeleteAssetModal.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from 'react';
+
+import {
+  Button,
+  Input,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  toast,
+} from '@massalabs/react-ui-kit';
+import { AxiosError } from 'axios';
+import { FiTrash2 } from 'react-icons/fi';
+import { useParams } from 'react-router-dom';
+
+import {
+  DeleteAssetsErrors,
+  assetDeleteErrors,
+  deleteConfirm,
+} from '@/const/assets/assets';
+import { useDelete } from '@/custom/api';
+import Intl from '@/i18n/i18n';
+import { IToken } from '@/models/AccountModel';
+
+export function DeleteAssetModal({ ...props }) {
+  const { tokenAddress, setModalOpen, refetch } = props;
+  const { nickname } = useParams();
+
+  const [deletePhrase, setDeletePhrase] = useState<string>('');
+  const [error, setError] = useState<DeleteAssetsErrors | null>(null);
+
+  const {
+    mutate: mutateDelete,
+    isSuccess: isSuccessDelete,
+    isError: isErrorDelete,
+    error: errorDelete,
+  } = useDelete<IToken[]>(
```

Comment:
in case of success, API endpoint returns no content, try with:

```suggestion
  } = useDelete<>(
```